crocodealer.cabal

```
@@ -29,6 +29,7 @@ library
 
   build-depends:       base-noprelude ^>= 4.12.0.0
                      , relude
```

For some reasons version bounds here were missing :thinking: 
```suggestion
                      , relude ^>= 0.5.0
```